src/Benchmarks.ClientJob/Benchmarks.ClientJob.csproj

```diff
@@ -4,12 +4,12 @@
 
   <PropertyGroup>
     <Description>Class to transfer data to/from the benchmark client.</Description>
-    <TargetFramework>netstandard1.0</TargetFramework>
```

Comment:
TBH I'm confused about this because I'm seeing everything being lifted to `netcoreapp2.0` in other repos, even things that were `netstandard1.*`. @Eilon what is the guidance here?